Nos/Models/CoreData/Event+CoreDataClass.swift

```diff
@@ -324,7 +324,48 @@ public class Event: NosManagedObject, VerifiableEvent {
         )
         return fetchRequest
     }
-    
+
+    /// A request for all events that responded to a specific note.
+    ///
+    /// - Parameter noteIdentifier: ID of the note to retrieve replies for.
+    ///
+    /// Intented to be used primarily to compute the number of replies and for
+    /// building a set of author avatars.
+    @nonobjc public class func replies(to noteIdentifier: String) -> FetchRequest<Event> {
```

Comment:
The method that was removed had `noteID: RawEventID` as a parameter -- is there a reason to do it differently here? I think ideally we'd be using the `RawEventID` wherever we can to signify that this is an event id and not just any String. I could be missing something, though.

```suggestion
    @nonobjc public class func replies(to noteID: RawEventID) -> FetchRequest<Event> {
```

Reply:
I renamed it to noteID and changed the String to RawEventID.